Enable using PostgreSQL connection strings

queue-tip opened the pull request:
This adds an argument for setting the DB_CONNECTION_STRING variable to enable using a PostgreSQL connection string with the bash script.

The `get_databases` function was changed to use the connection string's single database name when one is defined. Likewise, the `exec_query` function will use the connection string if it's defined.

I also added a missing argument to the `printf` statement used when no foreign tables exist on a database, and reformatted some of the if-then-else statements to be consistent with other uses in the file and make them easier to read.

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

pcelent reviewed (commented):
Have you tested and validated these modifications? I can spin up a test environment and will test before I approve, but I do intend to approve. Thank you for your contributions!

queue-tip commented:
I just tested it and made a few other changes as a result.

I've added input validation and changed the message when no endpoints file is provided.

Using a connection-string requires the use of either `awk` or `cut` to extract the host and database names from the connection string. In the end I decided to use `awk` because there's probably a higher probability of it being installed.

Is there anything else I should do?

queue-tip commented:
That commit I just pushed moves all validation to immediately after parsing all arguments. 

Let me know if you find any issues when you test, @pcelent .
